src/test/java/pl/edu/pw/elka/paprykaisalami/geeruh/IssuesPersistentTests.java

```diff
@@ -0,0 +1,60 @@
+package pl.edu.pw.elka.paprykaisalami.geeruh;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import pl.edu.pw.elka.paprykaisalami.geeruh.issues.domain.models.Description;
+import pl.edu.pw.elka.paprykaisalami.geeruh.issues.domain.models.IssueType;
+import pl.edu.pw.elka.paprykaisalami.geeruh.issues.domain.models.Summary;
+import pl.edu.pw.elka.paprykaisalami.geeruh.issues.domain.ports.IssueService;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@SpringBootTest
+public class IssuesPersistentTests {
+
+    @Autowired
+    private IssueService issueService;
+
+    @Test
+    void CreateIssue() {
+        var issue = issueService.create(IssueType.BUG, Summary.of("summary"), Description.of("desc"));
+
+        var issueFromGet = issueService.get(issue.getIssueId()).get();
+
+        assertThat(issue).isEqualTo(issueFromGet);
```

Comment:
```suggestion
        var issueFromGet = issueService.get(issue.getIssueId());

        assertThat(issueFromGet).hasValue(issue);
```
There is a rule of thumb in assertions that the expected value is on the right.

Also using `hasValue` prints you a pretty message if the optional is empty, instead of just throwing an error in line 23 at your face from using `get`.